node/pkg/node/node.go

```diff
@@ -335,7 +346,7 @@ func GuardianOptionAdminService(socketPath string, ethRpc *string, ethContract *
 				rpcMap,
 			)
 			if err != nil {
-				logger.Fatal("failed to create admin service socket", zap.Error(err))
+				return err
```

Comment:
Maybe the previous context was better?

```suggestion
				return fmt.Errorf("failed to create admin service socket: %w", err)
```